core/src/com/unciv/logic/UncivFiles.kt

```diff
@@ -328,12 +330,24 @@ class UncivFiles(
             // FileHandle is Gdx, but the class and JsonParser are not dependent on app initialization
             // In fact, at this point Gdx.app or Gdx.files are null but this still works.
             val file = FileHandle(base + File.separator + SETTINGS_FILE_NAME)
+
             return if (file.exists())
                 json().fromJsonFile(
                     GameSettings::class.java,
                     file
                 )
-            else GameSettings().apply { isFreshlyCreated = true }
+            else GameSettings().apply {
+                isFreshlyCreated = true
+                resolution = "1200x800" // By default Desktops should have a higher resolution
```

Comment:
I don't quite get it. Isn't this also called on android when the game is started the first time and not only on desktop?

Reply:
Nope

Reply:
I don't think so at least... Maybe I missed something?

Reply:
It does get called apparently 
but running your branch gives me an exception because awt is not available

```Kotlin
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.unciv.app, PID: 20238
    java.lang.NoClassDefFoundError: Failed resolution of: Ljava/awt/Toolkit;
        at com.unciv.logic.UncivFiles$Companion.getSettingsForPlatformLaunchers(UncivFiles.kt:343)
        at com.unciv.app.AndroidLauncher.onCreate(AndroidLauncher.kt:43)
        at android.app.Activity.performCreate(Activity.java:8000)
        at android.app.Activity.performCreate(Activity.java:7984)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1310)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3423)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3602)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2067)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7697)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:952)
     Caused by: java.lang.ClassNotFoundException: Didn't find class "java.awt.Toolkit" on path: DexPathList[[zip file "/data/app/~~ZqDBNMXWb3X93aaU1rrSbg==/com.unciv.app-ustl8wf_tTx0697aZafPyQ==/base.apk"],nativeLibraryDirectories=[/data/app/~~ZqDBNMXWb3X93aaU1rrSbg==/com.unciv.app-ustl8wf_tTx0697aZafPyQ==/lib/arm64, /data/app/~~ZqDBNMXWb3X93aaU1rrSbg==/com.unciv.app-ustl8wf_tTx0697aZafPyQ==/base.apk!/lib/arm64-v8a, /system/lib64, /system/system_ext/lib64, /system/product/lib64]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:207)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at com.unciv.logic.UncivFiles$Companion.getSettingsForPlatformLaunchers(UncivFiles.kt:343) 
        at com.unciv.app.AndroidLauncher.onCreate(AndroidLauncher.kt:43) 
        at android.app.Activity.performCreate(Activity.java:8000) 
        at android.app.Activity.performCreate(Activity.java:7984) 
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1310) 
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3423) 
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3602) 
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85) 
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2067) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:223) 
        at android.app.ActivityThread.main(ActivityThread.java:7697) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:952) 
W/System: A resource failed to call close. 
```